Update README with more setup information (#14)
CERT-Polska/karton-archive-extractor: README.md

First of all, make sure you have setup the core system: https://github.com/CERT-Polska/karton

In order to unpack all available formats you'll also need a few native dependencies that sflock relies on, so make sure you install them with:
```shell

> RUN sed -i 's/ main/ main non-free/' /etc/apt/sources.list \

Reviewer: Telling people to run sed on `/etc/apt/sources.list` is a bit shady :thinking:. But I guess technical people will know what it does and adjust accordingly?

Author: This command was copied directly from sflocks readme, seems kinda sketchy though. Should we omit it and just make the user worry about switching to non-free?

Reviewer: Maybe add `(copied from sflock readme)`. That way, when something breaks it's not technically out fault :duck:

$ karton-archive-extractor
```

## Running in Docker

> Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so remember to add the `SYS_PTRACE` capability if using Docker.

Reviewer: ```suggestion
Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so ensure that your container has `SYS_PTRACE` capability.
```

Maybe add example docker invocation :thinking: 

Also I don't think this is true - `SYS_PTRACE` is not necessary to trace your own children. Otherwise it would be impossible to use gdb without root, for example (non-root processes usually have no capabilities at all). Problem is somewhere else, probably :thinking:. Maybe rephrase that as a `debugging step`?
